### PR TITLE
[dv/otp_ctrl] Add coverage exclusions

### PR DESCRIPTION
**Pull request by cindychip**
1. Exclude otp_ctrl's FSM unreachable coverages
2. Exclude alert_sender module coverage

Here we did not perform a full UNR because design only reaches D2. There
will be some design changes.

Also I discussed with the designer and added a few sequence for
alert_sender auto sequence (PR #6089). Once this sequence is implemented, we will
come back and revisit the code coverage. But I believe alert_sender's
coverage should not prohibit IPs from reaching V2.

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el
Comment (weicaiyang) [inline hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el:89]: these UNR should be generated from the VCS UNR flow, right? 
since we should re-visit this again, can we add a TODO

**Review by msfschaffner — APPROVED** [review]

File: hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el
Comment (weicaiyang) [inline hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el:92]: this should be excluded if it's in prim_alert_sender as we exclude the entire prim_alert_sender, right?

File: hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el
Comment (cindychip) [reply hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el:89]: Sounds good. I will add a TODO for that. Yes it should generate from the UNR.

**Review by weicaiyang — APPROVED** [review]
as discussed offline, this LGTM

File: hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el
Comment (cindychip) [reply hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el:93]: Thanks weicai for taking the time to discuss the issue with me offline. What we found is that:
1. We are not able to use `*` notations to wildcard exclude alert_sender instances.
2. If we exclude module `prim_alert_sender`, it does not exclude `prim_diff_decode` automatically even though `prim_diff_decode` is inside `alert_sender`.
3. For now we will do this for all IPs to unblock coverage requirements. Once #6089 is landed, maybe we will have a better solution.

**Pull request merged**
